src/main/java/graphql/validation/AbstractRule.java

```diff
@@ -64,6 +74,10 @@ public ValidationContext getValidationContext() {
         return validationContext;
     }
 
```

Comment:
this is not a great type: List<Object>. Can we make it more clear?

Reply:
Well the spec says that once it boils down to an error it must be List<Object>

So I went straight to that since this is internal code